<rss xmlns:atom="http://www.w3.org/2005/Atom" version="2.0">
    <channel>
        <title>Algolia - 标签 - xiaobinqt 博客 - 技术改变生活</title>
        <link>https://www.xiaobinqt.cn/tags/algolia/</link>
        <description>Algolia - 标签 - xiaobinqt 博客 - 技术改变生活</description>
        <generator>Hugo -- gohugo.io</generator><language>zh-CN</language><managingEditor>xiaobinqt@163.com (xiaobinqt)</managingEditor>
            <webMaster>xiaobinqt@163.com (xiaobinqt)</webMaster><copyright>This work is licensed under a Creative Commons Attribution-NonCommercial 4.0 International License.</copyright><lastBuildDate>Mon, 28 Mar 2022 00:00:00 &#43;0000</lastBuildDate><atom:link href="https://www.xiaobinqt.cn/tags/algolia/" rel="self" type="application/rss+xml" /><item>
    <title>hugo algolia Unreachable hosts</title>
    <link>https://www.xiaobinqt.cn/hugo-algolia/</link>
    <pubDate>Mon, 28 Mar 2022 00:00:00 &#43;0000</pubDate><author>
        <name>xiaobinqt</name>
    </author><guid>https://www.xiaobinqt.cn/hugo-algolia/</guid>
    <description><![CDATA[最近在使用 hugo algolia 时，在 github actions 同步索引到 algolia 时总是出现这样的错误： action error list Unreachable hosts 我用的 action 插件是Algolia Index Uploader，找了半天发现是参数 algolia_index_id 写的]]></description>
</item></channel>
</rss>
